src/meta/api/src/schema_api_keys.rs

```diff
@@ -341,3 +354,76 @@ impl KVApiKey for TableStageFileNameIdent {
         })
     }
 }
+
+#[cfg(test)]
```

Comment:
Can we move the test block to a test file?

Reply:
A unit test to test private fields/methods has to be a nested `tests` mod.

A unit test that tests only one mod being a nested `tests` mod is also practical and efficient, IMHO.